src/displayapp/screens/CheckboxList.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <lvgl/lvgl.h>
+#include <cstdint>
+#include <memory>
+#include "displayapp/screens/Screen.h"
+#include "displayapp/Apps.h"
+#include "components/settings/Settings.h"
+
+#define MAXLISTITEMS 4
```

Comment:
we could make this define into a constexpr size_t and move it from the global namespace into the private CheckboxList one

Reply:
Done! However, since this constant is also used in the ctor(), I added it as a public field of the class.